packages/clerk-js/src/ui/components/UserProfile/utils.ts

```diff
@@ -16,12 +10,6 @@ export const currentSessionFirst = (id: string) => (a: IDable) => (a.id === id ?
 
 export const defaultFirst = (a: PhoneNumberResource) => (a.defaultSecondFactor ? -1 : 1);
 
-export function emailLinksEnabledForInstance(env: EnvironmentResource): boolean {
```

Comment:
I've moved this to `UserProfile` since it was easier to read it being placed in the same context as we changed requirements to include `enterprise_sso` 

Happy to move all of the utils included on this PR to this utils file here as well